src/lib/apiTypesV2/Subscription.cpp

```diff
@@ -63,7 +67,8 @@ std::string Notification::toJson()
   jh.addString("callback", this->callback);
   if (!this->throttling.isEmpty())
   {
-    jh.addRaw("throttling", this->throttling.render(JSON,"",false));
+    //jh.addRaw("throttling", this->throttling.render(JSON,"",false));
```

Comment:
Throttling::render seems to give back the entire json string already. Feeding it to addRaw also, gives you the ** "throttling": ** twice ...
Or, so it seems